dipy/testing/decorators.py

```diff
@@ -73,3 +73,18 @@ def test_with_xvfb(*args, **kwargs):
     # Plant it back in and return the new function:
     test_with_xvfb.__name__ = fname
     return test_with_xvfb if not is_windows else my_test
+
+
+def set_random_seed(func):
+    """Decorator to set equal seeds.
+
+    This will help us avoid random errors
+    that occur with low probabilitiy during testing.
+
+    """
+    def test_with_seed(*args, **kwargs):
+        np.random.seed(0)
```

Comment:
instead of zero, I would recommend to put a number., whatever it is.... something like this year (`np.random.seed(20223)`)